src/services/user.service.js

```diff
@@ -0,0 +1,36 @@
+const { User } = require('../models/User.model');
+const { generateRandomId } = require('../Random');
+
+const getAll = async () => {
+  return User.findAll();
+};
+
+const getById = async (id) => {
+  return User.findByPk(+id);
+};
+
+const updateUser = async ({ id, name }) => {
+  await User.update({ name }, { where: { id } });
+};
+
+const createUser = async (name) => {
+  const id = generateRandomId();
```

Comment:
In the 'createUser' function, you are generating a random 'id' for the user. If your database is set up to auto-increment the 'id' field, this could potentially lead to conflicts. Consider letting the database handle the 'id' generation, unless there is a specific need for this approach.